notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb

```diff
@@ -18,6 +18,8 @@
     "- [Original Pipeline Inference](#FastSAM-in-Ultralytics)\n",
```

Comment:
<p>Benchmark app does not always represent the performance boost accurately. For example in this case, FPS increased 4 times, however if you would run the model here, the performance boost would be less than 2x. </p><p> </p><p>Because of this I think it would be better to compare performance by running the real pipeline here, for example on calibration dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Done!</p>